sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -171,7 +171,7 @@ class APISpec extends Specification {
     }
 
     def setup() {
-        String fullTestName = specificationContext.getCurrentIteration().getName().replace(' ', '').toLowerCase()
+        String fullTestName = refactorName(specificationContext.getCurrentIteration().getName())
```

Comment:
This naming is slightly off. I took a look at this in the past and this is what I ended up using: https://github.com/alzimmermsft/azure-sdk-for-java/blob/AzStorage_UpgradeSpockCore/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy#L172

Reply:
I'll clean up name soon.